### PR TITLE
EDUCATOR-807 Fix Course Start date storing in Preview dashboard.

### PR DESCRIPTION
**Pull request by attiyaIshaque**
### Description
This PR fixes that filter of course start date is chronological not alphabetically.

### How to Test?
**Production**
https://prod-edx-discovery.edx.org/publisher/

- Click to *In preview* tab.
- Filter by Course start date.
- Sorting is alphabetical. 

**Sandbox**
- https://discovery-escalation-discovery.sandbox.edx.org/publisher/
- Click to *In preview* tab.
- Filter by Course start date.
- Sorting is chronological. 

You can use following credentials
_username : Attiya
Password: attiya_

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] @asadazam93 
- [x] @Rabia23 

FYI: @sstack22 

### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

File: course_discovery/templates/publisher/dashboard/_in_preview.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
                             {% endif %}
                             {{ course_run.owner_role_modified }}
                         </td>
-                        <td>
+                        <td data-order="{{ course_run.start|date:"Y-m-d" }}">
```
Comment (awaisdar001) [inline course_discovery/templates/publisher/dashboard/_in_preview.html:49]: @attiyaIshaque whats the documentation link?

**Review by awaisdar001 — CHANGES_REQUESTED** [review]

File: course_discovery/templates/publisher/dashboard/_in_preview.html (hunk 1 above)
Comment (attiyaIshaque) [reply course_discovery/templates/publisher/dashboard/_in_preview.html:49]: @awaisdar001 https://datatables.net/examples/advanced_init/html5-data-attributes.html

**Comment by Rabia23** [comment]
Looks good 👍

**Review by Rabia23 — APPROVED** [review]

**Review by asadazam93 — APPROVED** [review]
Looks Good

**Review by awaisdar001 — APPROVED** [review]

**Pull request merged**